platform/osx/export/export.cpp

```diff
@@ -425,6 +425,10 @@ Error EditorExportPlatformOSX::export_project(const Ref<EditorExportPreset> &p_p
 		}
 	}
 
+	if (!FileAccess::exists(p_path.get_base_dir())) {
+		return ERR_FILE_BAD_PATH;
```

Comment:
What about other platforms?

Reply:
I did only found this for JavaScript and OSX. Do you have any ideas where I can found it for the other platforms?

Reply:
It's in `platform/*/export/export.cpp` for each platform.

Reply:
Let me double check it.